openedx/core/djangoapps/notifications/views.py

```diff
@@ -286,9 +284,20 @@ def get(self, request):
 
             count_total += count
             count_by_app_name_dict[app_name] = count
-        # Return the unseen notifications count for the user and the unseen notifications count for each app name.
 
+        # Retrieve active course enrollments for the current user
+        learner_enrollments = CourseEnrollment.objects.filter(user=request.user, is_active=True)
```

Comment:
I think we do not need the complete CourseEnrollment objects here. We just need the course ids so we can use `.values_list` method here